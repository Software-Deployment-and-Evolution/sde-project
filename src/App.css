.App {
  text-align: center;
}

.App-logo {
  height: 40vmin;
  pointer-events: none;
}

@media (prefers-reduced-motion: no-preference) {
  .App-logo {
    animation: App-logo-spin infinite 20s linear;
  }
}

.App-header {
  background-color: #282c34;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}

.App-link {
  color: #61dafb;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}

.container {
  margin-left: 0;
  margin-right: 0;
  padding: 0;
  min-width: 100vw;
}

.display-2 {
  background-color: #61dafb;
  color: #282c34;
}

.display-5 {
  font-size: xx-large;
  padding-bottom: 1rem;
}

.form-check {
  padding-bottom: 2rem;
  font-size: large;
  font-style: italic;
}

.row {
  justify-content: space-between;
  padding-left: 8rem;
  padding-right: 8rem;
  padding-top: 2rem;
}

.card {
  padding-top: 1.5rem;
  box-shadow: rgba(50, 50, 93, 0.25) 0px 50px 100px -20px, rgba(0, 0, 0, 0.3) 0px 30px 60px -30px;
  background-color: lightyellow;
}