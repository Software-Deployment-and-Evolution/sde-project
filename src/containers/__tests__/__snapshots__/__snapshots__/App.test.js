import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import  App from '../../../../App'

describe('Snapshot Testing: App component', () => {
  it('renders without crashing with the save snapshot', () => {
    const wrapper = shallow( <App /> )
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})