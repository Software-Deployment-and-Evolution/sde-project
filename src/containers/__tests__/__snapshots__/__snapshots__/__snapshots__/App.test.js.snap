// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Testing: App component renders without crashing with the save snapshot 1`] = `
<div
  className="App"
>
  <WeekContainer />
</div>
`;
